build.sbt

```diff
@@ -200,42 +224,69 @@ lazy val monix = crossProject(JSPlatform, JVMPlatform)
   )
 
 lazy val monixJVM = monix.jvm
-lazy val monixJS = monix.js
+lazy val monixJS  = monix.js
 
-lazy val cli = crossProject(JVMPlatform)
```

Comment:
I had wrongfully analysed the problem. We need a specific IntellijRunner that dumps the test data in a teamCity specific format.